orga/app/components/routes/authenticated/campaigns/new-item.js

```diff
@@ -1,14 +1,23 @@
+import { inject as service } from '@ember/service';
 import { action, computed } from '@ember/object';
 import Component from '@ember/component';
 import { tracked } from '@glimmer/tracking';
 
 export default class NewItem extends Component {
+  @service currentUser;
 
-  campaign = null;
+  campaign = {};
   targetProfiles = null;
   wantIdPix = false;
   @tracked isCampaignGoalAssessment = null;
 
+  init() {
+    super.init(...arguments);
+    if (!this.currentUser.organization.canCollectProfiles) {
```

Comment:
```suggestion
 this.isTypeProfilesCollection = !this.currentUser.organization.canCollectProfiles
```
C'est pas une computed  property du coup ?

Reply:
En fait tu es obligatoirement en campaign de type assessment si tu ne peux pas collecter les profiles mais l'inverse n'est pas vrai.

Reply:
Parce que dans le cas où l'utilisateur peut collecter des profils, il fait son choix entre les deux types sans rien de pré-sélectionner.